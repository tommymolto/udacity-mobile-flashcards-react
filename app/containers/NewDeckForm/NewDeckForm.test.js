/* global describe, it, expect */

import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import storeConfig from '../../store/storeConfig';
import NewDeckForm from './NewDeckForm';

describe('NewDeckForm', () => {
  it('renders correctly', () => {
    const navigation = {
      state: {
        params: {
          title: 'React',
          questions: 2,
        },
      },
    };
    const tree = renderer
      .create(<Provider store={storeConfig}>
        <NewDeckForm content={[]} navigation={navigation} />
              </Provider>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
