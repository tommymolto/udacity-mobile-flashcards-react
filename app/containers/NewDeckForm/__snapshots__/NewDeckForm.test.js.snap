// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewDeckForm renders correctly 1`] = `
<View>
  <View
    style={
      Array [
        Object {
          "paddingHorizontal": 15,
        },
        undefined,
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "fontSize": 16,
            "paddingTop": 10,
          },
          undefined,
        ]
      }
    >
      What is the title of your new deck?
    </Text>
    <TextInput
      allowFontScaling={true}
      blurOnSubmit={true}
      keyboardType="default"
      maxLength={50}
      onChangeText={[Function]}
      placeholder="Deck title"
      style={
        Object {
          "height": 40,
        }
      }
    />
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "backgroundColor": "#05AE54",
            "color": "#ffffff",
            "fontSize": 22,
            "height": 45,
            "marginLeft": 40,
            "marginRight": 40,
            "marginVertical": 10,
            "paddingBottom": 10,
            "paddingTop": 10,
            "textAlign": "center",
          },
          undefined,
        ]
      }
    >
      Submit
    </Text>
  </View>
</View>
`;
